packages/bundle-source/src/is-entrypoint.js

```diff
@@ -0,0 +1,9 @@
+// Detect if this is run as a script.
+import url from 'url';
+import process from 'process';
+import fs from 'fs';
+
+// FIXME: Should maybe be exported by '@endo/something'?
```

Comment:
I’m fine with inlining the idiom for now, now that this establishes what the idiom is.

Reply:
so do I delete this comment? or leave it? or change it?

Reply:
No change requested.